config/initializers/assets.rb

```diff
@@ -1,7 +1,7 @@
 # Be sure to restart your server when you modify this file.
```

Comment:
- [ ] It would be better if we add the `sqlite3` gem to our gemfile so that we can properly use both `postgres` and `sqlite3` in our app. Currently, because of this conflict, I'm unable to run the app. Here is the screenshot of the terminal error I got trying to create and migrate

![Screenshot from 2022-09-21 08-59-04](https://user-images.githubusercontent.com/46388113/191462538-983ebba8-0f2b-473a-8a3c-da432e407405.png)